content/_authors/anna-livia.gomart.md

```diff
@@ -1,14 +1,19 @@
 ---
 fullname: Anna-Livia Gomart
-role: Développeuse
+role: Chief Product Officer Tech BetaGouv
 github: Anna-Livia
 avatar: 'https://avatars3.githubusercontent.com/Anna-Livia?s=600'
 link: 'https://www.anna-livia.com/'
 missions:
   - employer: octo
     end: 2021-03-31
     start: 2017-05-18
+    end: 2021-03-12
     status: independent
+- start: 2021-03-15
```

Comment:
Il semble que les checks échouent à cause de l'indentation de ce tiret - pour que le parser YAML s'en dépatouille il doit être aligné avec les autres de la liste.

Reply:
Aussi je te recommande par souci d'homogénéité de lister les champs de chaque objet dans le même ordre; commencer par `employer`, puis `status`, puis `start`, puis `end` serait assez logique. Mais c'est purement esthétique et tu n'en as pas besoin pour que les checks passent.

Reply:
Et aussi, welcome et félicitations. :)